packages/headless/src/features/question-answering/question-answering-state.ts

```diff
@@ -0,0 +1,20 @@
+export interface QuestionAnsweringState {
+  /**
+   * Determines if the snippet is liked, or upvoted by the end user.
+   */
+  liked: boolean;
+  /**
+   * Determines if the snippet is disliked, or downvoted by the end user.
+   */
+  disliked: boolean;
```

Comment:
Can something be both liked and disliked at the same time? If not, perhaps we should use an enum here, especially in the controller.